PostHog/Replay/PostHogReplayIntegration.swift

```diff
@@ -423,39 +429,40 @@
             }
 
             if let theSwitch = view as? UISwitch {
+                isSensitive = isSwitchSensitive(theSwitch)
                 wireframe.type = "input"
                 wireframe.inputType = "toggle"
                 wireframe.checked = theSwitch.isOn
                 if #available(iOS 14.0, *) {
                     if let text = theSwitch.title {
-                        wireframe.label = isSwitchSensitive(theSwitch) ? text.mask() : text
+                        wireframe.label = isSensitive || isParentSensitive ? text.mask() : text
                     }
                 }
             }
 
             if let imageView = view as? UIImageView {
+                isSensitive = isImageViewSensitive(imageView)
                 wireframe.type = "image"
                 if let image = imageView.image {
-                    if !isImageViewSensitive(imageView) {
+                    if !isSensitive, !isParentSensitive {
```

Comment:
Not sure this is correct as `and`? since all others are `or`.

Reply:
the other thing is that `isParentSensitive` is mostly if not all, about masking text, but in this case, the image will be or not masked based on that config too.

Reply:
`findMaskableWidgets` only passes the `maskChildren` flag if the view is `view.isNoCapture()`, I don't pass `maskChildren` if it's sensitive in text or image.
Every time I find a view sensitive in text or image, I return early, So I don't run into any of those problems.
I know you chose to implement it in a slightly different way, but take another look in `findMaskableWidgets` and learn from it.

Reply:
I pointed to the method implementation, plus gave a few ideas, but the current implementation is different.

Reply:
discussed offline, since this is out of scope, we will follow up on another PR, and get this bug fix merged first.